core/src/luigi-config/customResources/destinationRules.js

```diff
@@ -0,0 +1,78 @@
+export const destinationRules = {
+  details: {
+    header: [],
+    body: [
+      {
+        name: 'references',
+        widget: 'Panel',
+        children: [
+          {
+            source: 'spec.host',
+            type: 'string',
+            name: 'Host',
+          },
+          {
+            source: 'spec.exportTo',
+            widget: 'JoinedArray',
+            separator: ', ',
+          },
+        ],
+      },
+      {
+        source: 'spec.trafficPolicy',
+        name: 'spec.trafficPolicy',
+        widget: 'CodeViewer',
```

Comment:
please use other widgets to make the new view similar to the old one

Reply:
it will be a PR https://github.com/kyma-project/busola/pull/1775